kvrocks.conf

```diff
@@ -329,6 +329,13 @@ json-max-nesting-depth 1024
 # Default: json
 json-storage-format json
 
+# The write buffer size of minor column families those infrequently be used
+# including pubsub, propagte, zset_score, stream and search. 
+# It will reduce the memory usage in some scenarios.
+#
+# Default: 64 MB (same with rocksdb.write_buffer_size)
+minor-columns-write-buffer-size 64
```

Comment:
I think by default it can be 0, which means its value as same as `rocksdb.write_buffer_size`.